### PR TITLE
bpo-41343: Convert methods of complex to Argument Clinic

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue41343

### CODE REVIEW

File: Misc/NEWS.d/next/Core and Builtins/2020-07-19-23-54-52.bpo-41343.IR0Zy-.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Convert methods of :class:`complex` to Argument Clinic. Patch by Dong-hee
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Core and Builtins/2020-07-19-23-54-52.bpo-41343.IR0Zy-.rst]: We usually do not add NEWS entries when convert code to Argument Clinic. And the performance effect is insignificant in this case.

File: Objects/complexobject.c [hunk 2]
```diff
@@ -684,46 +684,58 @@ complex_float(PyObject *v)
     return NULL;
 }
 
+/*[clinic input]
+complex.conjugate
+
+complex.conjugate() -> complex
```
Comment (serhiy-storchaka) [inline Objects/complexobject.c]: How the output of help() looks now? Is not the signature is duplicated?

File: Objects/complexobject.c [hunk 3]
```diff
@@ -684,46 +684,58 @@ complex_float(PyObject *v)
...
-"\n"
-"Convert to a string according to format_spec.");
+
+/*[clinic input]
+complex.__format__
+
+    format_spec: unicode
+    /
+
+complex.__format__() -> str
```
Comment (serhiy-storchaka) [inline Objects/complexobject.c]: Same as above.

File: Objects/complexobject.c (hunk 2 above)
Comment (corona10) [reply Objects/complexobject.c]: ## Master
```
Help on built-in function conjugate:

conjugate(...) method of builtins.complex instance
    complex.conjugate() -> complex

    Return the complex conjugate of its argument. (3-4j).conjugate() == 3+4j.
```

## PR
```
Help on built-in function conjugate:

conjugate() method of builtins.complex instance
    complex.conjugate() -> complex

    Return the complex conjugate of its argument. (3-4j).conjugate() == 3+4j.
````

File: Objects/complexobject.c (hunk 3 above)
Comment (corona10) [reply Objects/complexobject.c]: ## Master
```
Help on built-in function __format__:

__format__(...) method of builtins.complex instance
    complex.__format__() -> str

    Convert to a string according to format_spec.
```
## PR
```
Help on built-in function __format__:

__format__(format_spec, /) method of builtins.complex instance
    complex.__format__() -> str

    Convert to a string according to format_spec.
```

**Review by serhiy-storchaka — COMMENTED** [review]
Please remove duplicated signatures. Compare with `help(int)` and `help(float)`.

**Comment by corona10** [comment]
@serhiy-storchaka 
> Please remove duplicated signatures. Compare with help(int) and help(float).

Updated!

**Review by serhiy-storchaka — APPROVED** [review]
:+1:

**Pull request merged**